src/Tests/Questions.test.js

```diff
@@ -1,9 +1,13 @@
+import { mount, render, shallow } from 'enzyme';
 import React from 'react';
+import QandA from '../QandA/QandA.jsx';
 
 
-describe('Questions', () => {
-  it('Should render two labels per question', () =>{
-    const foo = true;
-    expect(foo).toBe(true);
+describe('App', () => {
+  it('Should render to the page', () =>{
+    const component = mount(
+      <QandA />
+    );
+    expect(component.state().questions).toHaveLength(0);
```

Comment:
These enzyme tests seem to be pretty self explanatory, appreciate you taking the time to setup